src/ocl/gcn_asm_utils.cpp

```diff
@@ -239,7 +244,10 @@ static void AmdgcnAssembleQuiet(const std::string& source, const std::string& pa
 #ifdef __linux__
     std::stringstream clang_stdout_unused;
     const auto clang_path = GetGcnAssemblerPath();
-    const auto args       = " -x assembler -target amdgcn--amdhsa " + params + " " + source +
+    const auto args       = " -x assembler -target amdgcn--amdhsa " +
```

Comment:
Changes is in this file are not necessary. Why:
- Currently the library supports only parts that have XNACK mode OFF by default
  - See https://llvm.org/docs/AMDGPUUsage.html#processors
- The default value of the target feature is automatically set to the default XNACK setting of the target.
  - https://llvm.org/docs/AMDGPUUsage.html#target-features says: "_The target features supported by each processor, and the default value used if not specified explicitly, is listed in AMDGPU Processors._"

Moreover, this breaks compatibility with older assemblers that do not support the `-m[no-]xnack` option.

I am going to fix this right away.